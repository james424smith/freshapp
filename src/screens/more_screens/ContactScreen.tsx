import React, { useState, useEffect } from "react";
import { ScrollView, RefreshControl } from "react-native";
import { useSelector, useDispatch } from "react-redux";
import {
  getContactDetails,
  setRefreshingContactDetails,
} from "../../redux/actions";
import {
  CREW_MANAGER,
  PORT_AGENT,
  MANNING_AGENT,
} from "../../constants/contactConstants";
import { ContactType, Contact } from "../../interfaces";
import ContactScreenTopMenu from "../../components/contact/ContactScreenTopMenu";
import CrewManagerContactDetails from "../../components/contact/CrewManagerContactDetails";
import ManningAgentContactDetails from "../../components/contact/ManningAgentContactDetails";
import PortAgentContactDetails from "../../components/contact/PortAgentContactDetails";
import styles from "../../styles/more_screens/ContactsStyles";

import { IRootReducerType } from "../../redux/reducers";

const ContactScreen = () => {
  const [selectedType, setSelectedType] = useState<ContactType>(CREW_MANAGER);

  const contacts = useSelector<IRootReducerType, Contact>(
    ({ contactDetails }) => contactDetails.contactDetails
  );
  const isLoading = useSelector<IRootReducerType, boolean>(
    ({ contactDetails }) => contactDetails.loader
  );
  const dispatch = useDispatch();

  useEffect(() => {
    dispatch(getContactDetails());
  }, [dispatch]);

  const handleRefresh = () => {
    dispatch(setRefreshingContactDetails(true));
    dispatch(getContactDetails());
  };

  const handleSelection = (type: ContactType) => {
    setSelectedType(type);
  };

  const renderSelectedContactDetails = (selectedOption: string) => {
    switch (selectedOption) {
      case CREW_MANAGER:
        return (
          <CrewManagerContactDetails
            crewingManagerDetails={contacts?.crewingManager?.crewingManager}
            photoSmall={contacts?.crewingManager?.photoSmall}
          />
        );
      case MANNING_AGENT:
        return (
          <ManningAgentContactDetails
            manningAgentDetails={contacts?.manningAgent}
          />
        );
      case PORT_AGENT:
        return (
          <PortAgentContactDetails portAgentDetails={contacts?.portAgent} />
        );
      default:
        throw Error("Component path not found");
    }
  };
  return (
    <ScrollView
      testID={"scrollview"}
      style={styles.root}
      refreshControl={
        <RefreshControl refreshing={isLoading} onRefresh={handleRefresh} />
      }
    >
      <ContactScreenTopMenu
        handleSelection={handleSelection}
        selectedType={selectedType}
      />
      <ScrollView style={styles.scrollViewContainer}>
        {renderSelectedContactDetails(selectedType)}
      </ScrollView>
    </ScrollView>
  );
};

export default ContactScreen;
