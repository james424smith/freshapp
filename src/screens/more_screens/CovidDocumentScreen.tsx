import React, { useEffect } from "react";
import { ScrollView, View, StyleSheet } from "react-native";
import PdfRender from "../../components/PdfRender";
import { useSelector, useDispatch } from "react-redux";
import { getCovidDocument } from "../../redux/actions";
import { IRootReducerType } from "../../redux/reducers";
import LoadingScreen from "../../components/LoadingScreen";

const logoSize = 100;

const styles = StyleSheet.create({
  root: { flex: 1 },
  scrollViewContainer: {
    flexGrow: 1,
  },
});

const CovidDocumentScreen = () => {
  const covidDocumentFile = useSelector<IRootReducerType, string>(
    ({ covidDocument }) => covidDocument.covidDocument
  );

  const isLoading = useSelector<IRootReducerType, boolean>(
    ({ covidDocument }) => covidDocument.loader
  );

  const dispatch = useDispatch();

  useEffect(() => {
    dispatch(getCovidDocument());
  }, [dispatch]);

  const pdf = covidDocumentFile;

  if (isLoading) {
    return (
      <View style={styles.root}>
        <LoadingScreen logoWidthSize={logoSize} logoHeightSize={logoSize} />
      </View>
    );
  }

  return (
    <ScrollView contentContainerStyle={styles.scrollViewContainer}>
      {pdf ? <PdfRender document={pdf} resourceType={"base64"} /> : <View />}
    </ScrollView>
  );
};

export default CovidDocumentScreen;
