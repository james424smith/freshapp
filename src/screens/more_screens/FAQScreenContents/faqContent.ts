const email = "mailto:feedback.app@marlowgroup.com";

export const assignmentContent = [
  {
    id: 0,
    question: "I cannot see my flights / details of next assignment?",
    answer:
      "Please contact your local manning agency or crew manager.  They will update your database file and then you will see these changes in the App.",
    opened: false,
  },
  {
    id: 1,
    question: "I cannot see previous payroll / working clothes records?",
    answer:
      "Presently the App only shows these records for your present / current assignment.  Historical records are not shown",
    opened: false,
  },
  {
    id: 2,
    question: "How do I see the location of the vessel I am joining?",
    answer:
      "Go to the ‘Assignment screen’ and click on the map icon (top right)",
    opened: false,
  },
];

export const generalContent = [
  {
    id: 0,
    question: "I have a suggestion to improve the App",
    answer:
      "We are happy to hear all feedback. Please contact our support team for further assistance (feedback.app@marlowgroup.com) or through our support page in our application.",
    opened: false,
    url: email,
  },
  {
    id: 1,
    question: "Where do I see Marlow's job openings? Can I apply online?",
    answer:
      "You can always visit the link below to see Marlow's open vacancies and follow the application procedures. \n  https://marlow-navigation.com/en/marlow-careers.asp'",
    opened: false,
    url: "https://marlow-navigation.com/en/marlow-careers.asp",
  },
];

export const loginAndPasswordContent = [
  {
    id: 0,
    question: "The temporary password I received by email does not work",
    answer:
      "Firstly, you need to ensure that all 8 characters are copied / pasted. \n\nSecondly, the temp password is only valid for 14 days after issuance. \n\nIf the password has expired please contact our support team for further assistance (feedback.app@marlowgroup.com). \n\nThey will reset you account in order that you can start the process again from the beginning. ",
    opened: false,
    url: email,
  },
  {
    id: 1,
    question: "I cannot create a personal password. It is not accepted.",
    answer:
      "Please enter your own password twice ensuring that: \n a. It is a least 8 characters  \n b. It contains one capital letter  \n c. It contains one symbol (!@#$%^&*) \n d. Both entries are identical",
    opened: false,
  },
  {
    id: 2,
    question: "I have forgotten my personal password",
    answer:
      "Please use the ‘Forgot Password’ option on the login page. You will then receive an email with further guidance",
    opened: false,
  },
  {
    id: 3,
    question:
      "I used the ‘Forgot Password’ option but did not receive an email",
    answer:
      "Please contact our support team for further assistance (feedback.app@marlowgroup.com)",
    opened: false,
    url: email,
  },
  {
    id: 4,
    question: "How can I enable biometric login (finger print / face ID)?",
    answer:
      "a. You need to login into the application using your credentials. \nb. Set your security passcode \nc. Select More from the menu bar. \nd. Select the Settings tab. \ne. Swipe button to the right to Enable Biometrics. \nf. Use biometric confirmation to enable this feature. \n Please note that Biometric security settings should be configured on your smartphone to be able to activate on the application.",

    opened: false,
  },
];

export const personalInfoContent = [
  {
    id: 0,
    question: "Is my Crew Portal and CrewCompanion account linked?",
    answer:
      "a. No. Both systems are separate for each other.  For both systems you firstly need to register and create a personal password \n\nb. Crew Portal Registration guide",
    opened: false,
  },
  {
    id: 1,
    question: "The details in my account are not correct / need updating?",
    answer:
      "Please contact your local manning agency or crew manager.  They will update your database file and then you will see these changes in the App.",
    opened: false,
  },
  {
    id: 2,
    question: "I cannot see previous payroll / working clothes records?",
    answer:
      "Yes. For each document entry you will see switch.  Slide this switch from left to right.  The switch will change to blue.  The image of the document has been download and can be viewed in the App when offline.",
    opened: false,
  },
];

export const registrationContent = [
  {
    id: 0,
    question:
      "I am trying to create an account and I receive a message that my email address does not match the entry in my file.",
    answer:
      "Please contact your local manning agency or crew manager. They will update the email address in your database file.",
    opened: false,
  },
  {
    id: 1,
    question:
      "During registration I did not receive an email with the subject “Your Marlow CrewCompanion Account Registration”",
    answer:
      "Please also check Spam and Deleted items folders in case the email was automatically filed in these locations. If you still cannot find the mail please contact our Support Team (feedback.app@marlowgroup.com)",
    opened: false,
    url: email,
  },
];

export const technicalContant = [
  {
    id: 0,
    question: "How do I update the App?",
    answer:
      "a. Android users can update via the Google Play store. \nb. iOS users can update by downloading the installation file via the Marlow Crew Portal",
    opened: false,
  },
  {
    id: 1,
    question: "I cannot see anyone in the Chat feature? ",
    answer:
      "a. Please make sure that you have enabled GPS / location services in the settings screen \n\nb. Please make sure you have activated the service for the required time (30 mins to 5.5 hours) \n\nc. Please note that you will only be able to view other Marlow seafarers that have activated the chat feature and are within 30km radius range",
    opened: false,
  },
  {
    id: 2,
    question: "I have an iPhone. Why can’t I download via the Apple App Store?",
    answer:
      "Apple do not allow Apps for internal employees to be listed in their store. You can download and install the App via the Marlow Crew Portal",
    opened: false,
  },
  {
    id: 3,
    question: "I have an iPhone. How do I enable ‘dark mode’ for the App?",
    answer: "Go to ‘settings’ and then ‘display and brightness’",
    opened: false,
  },
  {
    id: 4,
    question: "How can I enable biometric login (finger print / face ID)?",
    answer:
      "a. You need to login into the application using your credentials. \nb. Set your security passcode \nc. Select More from the menu bar. \nd. Select the Settings tab. \ne. Swipe button to the right to Enable Biometrics.\nf. Use biometric confirmation to enable this feature. \n Please note that Biometric security settings should be configured on your smartphone to be able to activate on the application.",

    opened: false,
  },
  {
    id: 5,
    question:
      "I cannot share my documents on my Huawei/Xiaomi device. What can I do?",
    answer:
      "a. Go to Settings -> Apps \nb. Select Default apps \nc. Go to More defaults. \nd. Select Gallery. \ne. Swipe button to the right to Enable Biometrics. \nf. Use biometric confirmation to enable this feature. \n Please note that Biometric security settings should be configured on your smartphone to be able to activate on the application.",
    opened: false,
  },
];
