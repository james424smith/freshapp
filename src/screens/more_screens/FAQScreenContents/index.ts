import Registration from "./RegistrationTab";
import LoginandPassword from "./LoginAndPasswordTab";
import PersonalInformation from "./PersonalInfoTab";
import Assignment from "./AssignmentTabContent";
import Technical from "./TechnicalTab";
import General from "./GeneralTab";

export {
  Registration,
  LoginandPassword,
  PersonalInformation,
  Assignment,
  Technical,
  General,
};
