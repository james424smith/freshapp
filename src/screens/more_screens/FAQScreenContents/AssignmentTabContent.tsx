import React from "react";
import { View } from "react-native";
import { assignmentContent } from "./faqContent";
import FaqView from "../../../components/FaqView";
type Props = {
  selectedQuestion?: number;
};

const Assignment = (props: Props) => {
  return (
    <View>
      <FaqView
        content={assignmentContent}
        selectedQuestion={props.selectedQuestion}
      />
    </View>
  );
};

export default Assignment;
