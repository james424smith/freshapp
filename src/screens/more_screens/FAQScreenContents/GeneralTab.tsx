import React from "react";
import { View } from "react-native";
import FaqView from "../../../components/FaqView";
import { generalContent } from "./faqContent";
type Props = {
  selectedQuestion?: number;
};

const General = (props: Props) => {
  return (
    <View>
      <FaqView
        content={generalContent}
        selectedQuestion={props.selectedQuestion}
      />
    </View>
  );
};

export default General;
