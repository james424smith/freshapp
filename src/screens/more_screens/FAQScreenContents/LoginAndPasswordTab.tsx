import React from "react";
import { View } from "react-native";
import { loginAndPasswordContent } from "./faqContent";
import FaqView from "../../../components/FaqView";
type Props = {
  selectedQuestion?: number;
};

const LoginAndPassword = (props: Props) => {
  return (
    <View>
      <FaqView
        content={loginAndPasswordContent}
        selectedQuestion={props.selectedQuestion}
      />
    </View>
  );
};

export default LoginAndPassword;
