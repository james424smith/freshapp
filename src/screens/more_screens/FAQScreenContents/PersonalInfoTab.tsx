import React from "react";
import { View } from "react-native";
import FaqView from "../../../components/FaqView";
import { personalInfoContent } from "./faqContent";

type Props = {
  selectedQuestion?: number;
};

const PersonalInformation = (props: Props) => {
  return (
    <View>
      <FaqView
        content={personalInfoContent}
        selectedQuestion={props.selectedQuestion}
      />
    </View>
  );
};

export default PersonalInformation;
