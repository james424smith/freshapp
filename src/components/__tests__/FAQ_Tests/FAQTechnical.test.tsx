import React from "react";
import { render, cleanup, RenderAPI } from "@testing-library/react-native";
import { Technical } from "../../../screens/more_screens/FAQScreenContents";
import "@testing-library/jest-native/extend-expect";

describe("FAQ Screen UI test", () => {
  let wrapper: RenderAPI;

  beforeEach(() => {
    wrapper = render(<Technical />);
  });

  afterEach(() => {
    cleanup();
    jest.clearAllMocks();
  });

  it("Should render the screen as UI", () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
