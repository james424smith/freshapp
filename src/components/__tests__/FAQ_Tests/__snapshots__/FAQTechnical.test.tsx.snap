// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQ Screen UI test Should render the screen as UI 1`] = `
<View>
  <RCTScrollView
    data={
      Array [
        Object {
          "answer": "a. Android users can update via the Google Play store. 
b. iOS users can update by downloading the installation file via the Marlow Crew Portal",
          "id": 0,
          "opened": false,
          "question": "How do I update the App?",
        },
        Object {
          "answer": "a. Please make sure that you have enabled GPS / location services in the settings screen 

b. Please make sure you have activated the service for the required time (30 mins to 5.5 hours) 

c. Please note that you will only be able to view other Marlow seafarers that have activated the chat feature and are within 30km radius range",
          "id": 1,
          "opened": false,
          "question": "I cannot see anyone in the Chat feature? ",
        },
        Object {
          "answer": "Apple do not allow Apps for internal employees to be listed in their store. You can download and install the App via the Marlow Crew Portal",
          "id": 2,
          "opened": false,
          "question": "I have an iPhone. Why can’t I download via the Apple App Store?",
        },
        Object {
          "answer": "Go to ‘settings’ and then ‘display and brightness’",
          "id": 3,
          "opened": false,
          "question": "I have an iPhone. How do I enable ‘dark mode’ for the App?",
        },
        Object {
          "answer": "a. You need to login into the application using your credentials. 
b. Set your security passcode 
c. Select More from the menu bar. 
d. Select the Settings tab. 
e. Swipe button to the right to Enable Biometrics.
f. Use biometric confirmation to enable this feature. 
 Please note that Biometric security settings should be configured on your smartphone to be able to activate on the application.",
          "id": 4,
          "opened": false,
          "question": "How can I enable biometric login (finger print / face ID)?",
        },
        Object {
          "answer": "a. Go to Settings -> Apps 
b. Select Default apps 
c. Go to More defaults. 
d. Select Gallery. 
e. Swipe button to the right to Enable Biometrics. 
f. Use biometric confirmation to enable this feature. 
 Please note that Biometric security settings should be configured on your smartphone to be able to activate on the application.",
          "id": 5,
          "opened": false,
          "question": "I cannot share my documents on my Huawei/Xiaomi device. What can I do?",
        },
      ]
    }
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    getItemLayout={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View>
      <View
        style={null}
      >
        <RNGestureHandlerButton
          collapsable={false}
          onGestureEvent={[Function]}
          onGestureHandlerEvent={[Function]}
          onGestureHandlerStateChange={[Function]}
          onHandlerStateChange={[Function]}
          rippleColor={0}
          testID="faq-button-0"
        >
          <View
            accessible={true}
            style={
              Object {
                "borderBottomColor": "#C5C5C5",
                "borderBottomWidth": 0.3,
                "opacity": 1,
                "paddingVertical": 20,
              }
            }
          >
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "marginLeft": 22.5,
                  "marginRight": 30,
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#2A2E43",
                      "fontFamily": "Roboto-Regular",
                    },
                    Object {
                      "color": undefined,
                      "fontSize": 14,
                      "fontWeight": "bold",
                      "width": 600,
                    },
                  ]
                }
              >
                How do I update the App?
              </Text>
              <View
                style={
                  Object {
                    "overflow": "hidden",
                  }
                }
              >
                <View>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "justifyContent": "center",
                      }
                    }
                  >
                    <Text
                      allowFontScaling={false}
                      style={
                        Array [
                          Object {
                            "color": "#8E9397",
                            "fontSize": 30,
                          },
                          Object {
                            "backgroundColor": "transparent",
                          },
                          Object {
                            "fontFamily": "Material Icons",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                          Object {},
                        ]
                      }
                      testID="iconIcon"
                    >
                      
                    </Text>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </RNGestureHandlerButton>
      </View>
      <View
        style={null}
      >
        <RNGestureHandlerButton
          collapsable={false}
          onGestureEvent={[Function]}
          onGestureHandlerEvent={[Function]}
          onGestureHandlerStateChange={[Function]}
          onHandlerStateChange={[Function]}
          rippleColor={0}
          testID="faq-button-1"
        >
          <View
            accessible={true}
            style={
              Object {
                "borderBottomColor": "#C5C5C5",
                "borderBottomWidth": 0.3,
                "opacity": 1,
                "paddingVertical": 20,
              }
            }
          >
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "marginLeft": 22.5,
                  "marginRight": 30,
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#2A2E43",
                      "fontFamily": "Roboto-Regular",
                    },
                    Object {
                      "color": undefined,
                      "fontSize": 14,
                      "fontWeight": "bold",
                      "width": 600,
                    },
                  ]
                }
              >
                I cannot see anyone in the Chat feature? 
              </Text>
              <View
                style={
                  Object {
                    "overflow": "hidden",
                  }
                }
              >
                <View>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "justifyContent": "center",
                      }
                    }
                  >
                    <Text
                      allowFontScaling={false}
                      style={
                        Array [
                          Object {
                            "color": "#8E9397",
                            "fontSize": 30,
                          },
                          Object {
                            "backgroundColor": "transparent",
                          },
                          Object {
                            "fontFamily": "Material Icons",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                          Object {},
                        ]
                      }
                      testID="iconIcon"
                    >
                      
                    </Text>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </RNGestureHandlerButton>
      </View>
      <View
        style={null}
      >
        <RNGestureHandlerButton
          collapsable={false}
          onGestureEvent={[Function]}
          onGestureHandlerEvent={[Function]}
          onGestureHandlerStateChange={[Function]}
          onHandlerStateChange={[Function]}
          rippleColor={0}
          testID="faq-button-2"
        >
          <View
            accessible={true}
            style={
              Object {
                "borderBottomColor": "#C5C5C5",
                "borderBottomWidth": 0.3,
                "opacity": 1,
                "paddingVertical": 20,
              }
            }
          >
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "marginLeft": 22.5,
                  "marginRight": 30,
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#2A2E43",
                      "fontFamily": "Roboto-Regular",
                    },
                    Object {
                      "color": undefined,
                      "fontSize": 14,
                      "fontWeight": "bold",
                      "width": 600,
                    },
                  ]
                }
              >
                I have an iPhone. Why can’t I download via the Apple App Store?
              </Text>
              <View
                style={
                  Object {
                    "overflow": "hidden",
                  }
                }
              >
                <View>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "justifyContent": "center",
                      }
                    }
                  >
                    <Text
                      allowFontScaling={false}
                      style={
                        Array [
                          Object {
                            "color": "#8E9397",
                            "fontSize": 30,
                          },
                          Object {
                            "backgroundColor": "transparent",
                          },
                          Object {
                            "fontFamily": "Material Icons",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                          Object {},
                        ]
                      }
                      testID="iconIcon"
                    >
                      
                    </Text>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </RNGestureHandlerButton>
      </View>
      <View
        style={null}
      >
        <RNGestureHandlerButton
          collapsable={false}
          onGestureEvent={[Function]}
          onGestureHandlerEvent={[Function]}
          onGestureHandlerStateChange={[Function]}
          onHandlerStateChange={[Function]}
          rippleColor={0}
          testID="faq-button-3"
        >
          <View
            accessible={true}
            style={
              Object {
                "borderBottomColor": "#C5C5C5",
                "borderBottomWidth": 0.3,
                "opacity": 1,
                "paddingVertical": 20,
              }
            }
          >
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "marginLeft": 22.5,
                  "marginRight": 30,
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#2A2E43",
                      "fontFamily": "Roboto-Regular",
                    },
                    Object {
                      "color": undefined,
                      "fontSize": 14,
                      "fontWeight": "bold",
                      "width": 600,
                    },
                  ]
                }
              >
                I have an iPhone. How do I enable ‘dark mode’ for the App?
              </Text>
              <View
                style={
                  Object {
                    "overflow": "hidden",
                  }
                }
              >
                <View>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "justifyContent": "center",
                      }
                    }
                  >
                    <Text
                      allowFontScaling={false}
                      style={
                        Array [
                          Object {
                            "color": "#8E9397",
                            "fontSize": 30,
                          },
                          Object {
                            "backgroundColor": "transparent",
                          },
                          Object {
                            "fontFamily": "Material Icons",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                          Object {},
                        ]
                      }
                      testID="iconIcon"
                    >
                      
                    </Text>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </RNGestureHandlerButton>
      </View>
      <View
        style={null}
      >
        <RNGestureHandlerButton
          collapsable={false}
          onGestureEvent={[Function]}
          onGestureHandlerEvent={[Function]}
          onGestureHandlerStateChange={[Function]}
          onHandlerStateChange={[Function]}
          rippleColor={0}
          testID="faq-button-4"
        >
          <View
            accessible={true}
            style={
              Object {
                "borderBottomColor": "#C5C5C5",
                "borderBottomWidth": 0.3,
                "opacity": 1,
                "paddingVertical": 20,
              }
            }
          >
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "marginLeft": 22.5,
                  "marginRight": 30,
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#2A2E43",
                      "fontFamily": "Roboto-Regular",
                    },
                    Object {
                      "color": undefined,
                      "fontSize": 14,
                      "fontWeight": "bold",
                      "width": 600,
                    },
                  ]
                }
              >
                How can I enable biometric login (finger print / face ID)?
              </Text>
              <View
                style={
                  Object {
                    "overflow": "hidden",
                  }
                }
              >
                <View>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "justifyContent": "center",
                      }
                    }
                  >
                    <Text
                      allowFontScaling={false}
                      style={
                        Array [
                          Object {
                            "color": "#8E9397",
                            "fontSize": 30,
                          },
                          Object {
                            "backgroundColor": "transparent",
                          },
                          Object {
                            "fontFamily": "Material Icons",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                          Object {},
                        ]
                      }
                      testID="iconIcon"
                    >
                      
                    </Text>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </RNGestureHandlerButton>
      </View>
      <View
        style={null}
      >
        <RNGestureHandlerButton
          collapsable={false}
          onGestureEvent={[Function]}
          onGestureHandlerEvent={[Function]}
          onGestureHandlerStateChange={[Function]}
          onHandlerStateChange={[Function]}
          rippleColor={0}
          testID="faq-button-5"
        >
          <View
            accessible={true}
            style={
              Object {
                "borderBottomColor": "#C5C5C5",
                "borderBottomWidth": 0.3,
                "opacity": 1,
                "paddingVertical": 20,
              }
            }
          >
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                  "marginLeft": 22.5,
                  "marginRight": 30,
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#2A2E43",
                      "fontFamily": "Roboto-Regular",
                    },
                    Object {
                      "color": undefined,
                      "fontSize": 14,
                      "fontWeight": "bold",
                      "width": 600,
                    },
                  ]
                }
              >
                I cannot share my documents on my Huawei/Xiaomi device. What can I do?
              </Text>
              <View
                style={
                  Object {
                    "overflow": "hidden",
                  }
                }
              >
                <View>
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "justifyContent": "center",
                      }
                    }
                  >
                    <Text
                      allowFontScaling={false}
                      style={
                        Array [
                          Object {
                            "color": "#8E9397",
                            "fontSize": 30,
                          },
                          Object {
                            "backgroundColor": "transparent",
                          },
                          Object {
                            "fontFamily": "Material Icons",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                          Object {},
                        ]
                      }
                      testID="iconIcon"
                    >
                      
                    </Text>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </RNGestureHandlerButton>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
